mobileChatExamples/androidChatExample/README.md

```diff
@@ -1,20 +1,25 @@
 # Android Native Chat Demo 📱
 
-A native android example app for building custom Amazon Connect Chat. This solution implements basic [ChatWidget](https://docs.aws.amazon.com/connect/latest/adminguide/add-chat-to-website.html) functionality and is capable of Interactive Messages.
+This is an example app on how to utilise [AmazonConnectChatAndroid](https://github.com/amazon-connect/amazon-connect-chat-android) SDK
```

Comment:
(sp) - utilize

Instead of AmazonConnectChatAndroid SDK, could you write:
Amazon Connect Chat SDK for Android